README.md

```diff
@@ -40,19 +40,27 @@ tasks with a single function call.
 
 * Want to understand the distributions for each DataFrame column? Use `plot`.
 
-<center><a href="https://sfu-db.github.io/dataprep/eda/introduction.html#analyzing-basic-characteristics-via-plot"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/plot(df).png"/></a></center>
+<center><a href="https://sfu-db.github.io/dataprep/eda/introduction.html#analyzing-basic-characteristics-via-plot"><img src="assets/plot(df).gif"/></a></center>
```

Comment:
Hi please use the absolute URL instead of the relative one. PyPI will not show things correctly with relative url.

Reply:
The GIF won't be displayed if I used the absolute URL here. I'm also curious.

Reply:
Change the URL to this https://github.com/dylanzxc/dataprep/raw/test/assets/plot(df).gif

Change it to back sfu-db before we merge.

Reply:
fixed

Reply:
Can you guys take a look at the issue with <center>? 

https://pypi.org/project/dataprep/

<img width="846" alt="Screen Shot 2020-07-14 at 10 47 42 PM" src="https://user-images.githubusercontent.com/1028970/87508464-4e1f0700-c624-11ea-8d78-9d8e1c6c067e.png">

Reply:
@dylanzxc Can you also remove the center HTML tag?

Reply:
removed. Do you know why the center HTML tag affects the pypi now? The reason is HTML5 removed <center>?

Reply:
I'm not sure. Maybe you can find some clue in their renderer? https://github.com/pypa/readme_renderer